src/Thorn_Calculus-Semantics-Synchronous.thy

```diff
@@ -145,6 +145,11 @@ proof -
     by force
 qed
 
+lemma repeated_receive_transition:
+  shows "A \<triangleright>\<^sup>\<infinity> x. \<P> x \<rightarrow>\<^sub>s\<lparr>A \<triangleright> \<star>\<^bsup>n\<^esup> X\<rparr> post_receive n X \<P> \<parallel> (A \<triangleright>\<^sup>\<infinity> x. \<P> x) \<guillemotleft> suffix n"
+  using receiving [where \<P> = "\<lambda>x. \<P> x \<parallel> A \<triangleright>\<^sup>\<infinity> x. \<P> x", unfolded post_receive_after_parallel]
+  by (subst repeated_receive_proper_def) force
```

Comment:
This whole lemma should appear before `transition_from_repeated_receive`, like, for example, `adapted_communication_transition` appears before `communication_transition_from_adapted`.

Reply:
Fixed in 061ffd5c4e78455d8638373a01fd8420fcf5fcae, please verify.

Reply:
Looks good.